src/controllers/expenses.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+const usersService = require('../services/users');
+
+const getAllFiltered = (req, res) => {
+  const expenses = expensesService.filterExpenses(req.query);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expensesService.getExpenseById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const add = (req, res) => {
+  const {
+    userId,
+    title,
+  } = req.body;
+
+  const checkOnUser = usersService.getUserById(userId);
+
+  if (!checkOnUser || !title) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpence = expensesService.createExpense(req.body);
+
+  res.statusCode = 201;
```

Comment:
Use 'res.status(201)' instead of 'res.statusCode = 201' for setting the status code.